Modules/_base64/src/lib.rs

```diff
@@ -133,12 +208,12 @@ pub unsafe extern "C" fn standard_b64encode(
 
     // Safe cast by Safety
     match standard_b64encode_impl(source) {
-        Ok(result) => result,
+        Ok(result) => PyRc::into_raw(result),
         Err(_) => ptr::null_mut(),
     }
 }
 
-fn standard_b64encode_impl(source: &PyObject) -> Result<*mut PyObject, ()> {
+fn standard_b64encode_impl(source: &PyObject) -> Result<PyRc, ()> {
```

Comment:
&PyObject means it guarantees the RC count is never changed in this logic. (not 100% because it can be changed in C side)
PyRc means it can be increased or decreased.